modules/git/repo_compare.go

```diff
@@ -295,6 +295,37 @@ func (repo *Repository) GetFilesChangedBetween(base, head string) ([]string, err
 	return split, err
 }
 
+// GetCommitFilesChanged get the changed file names of the specified commit
+func (repo *Repository) GetCommitFilesChanged(commitID string) ([]string, error) {
+	id, err := repo.ConvertToSHA1(commitID)
+	if err != nil {
+		return nil, err
+	}
+	commit, err := repo.getCommit(id)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
It seems inefficient, there was already a "commit", I do not think it needs to do a "getCommit" again.

I will try to make some changes.

Reply:
Yes, I noticed this problem but I don't have a good solution. 

Thanks for your improvements